src/setUnion.js

```diff
@@ -0,0 +1,12 @@
+export default function setUnion(a, b){
+  if (Array.isArray(a) && Array.isArray(b)){
+    for (var i = 0; i < b.length; i++){
+      if (a.includes(b[i]) === false){
```

Comment:
Since you are referencing `b[i]` twice, you might consider assigning to a variable. Also, naming it would make it easier to understand your code more quickly.
```js
var currentValue = b[i] 
```